src/endpoints/mex/mex.pair.service.ts

```diff
@@ -51,9 +56,13 @@ export class MexPairService {
     );
   }
 
-  async getMexPairsCount(): Promise<number> {
+  async getMexPairsCount(filter?: MexPairExchange): Promise<number> {
```

Comment:
please create a MexPairsFilter class to hold the exchange filter, so that it's in line with other functions and also can be easily extended